docs/installation/deployment/objecttypes-api/kubernetes.rst

```diff
@@ -22,9 +22,9 @@ Install the Helm chart with following commands:
 
 .. code:: shell
 
```

Comment:
In this file, we could maybe link to https://github.com/maykinmedia/charts/tree/main/charts/objecttypen as well (same for Objects API)